Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -351,6 +351,35 @@ - (void)resume
     [self.sessionReplay resume];
 }
 
+- (void)start
+{
+    if (self.sessionReplay != nil) {
+        if (self.sessionReplay.isFullSession == NO) {
+            [self.sessionReplay captureReplay];
+        }
+        return;
+    }
+
+    _startedAsFullSession = YES;
+    if (SentryDependencyContainer.sharedInstance.application.windows.count > 0) {
+        // If a window its already available start replay right away
+        [self startWithOptions:_replayOptions fullSession:YES];
+    } else {
+        // Wait for a scene to be available to started the replay
+        if (@available(iOS 13.0, tvOS 13.0, *)) {
+            [_notificationCenter addObserver:self
+                                    selector:@selector(newSceneActivate)
+                                        name:UISceneDidActivateNotification];
+        }
+    }
+}
+
+- (void)stop
+{
+    [self.sessionReplay pause];
```

Comment:
`l`: It's a bit confusing to me that this method here is named `pause`. It would be great to be consistent with the naming.

Reply:
Is this a typo? I saw that there was a new function `stop` added to SentrySessionReplay: https://github.com/getsentry/sentry-cocoa/pull/4414/files#diff-7e90bc4012112a489d229dd525a4bb5c21ee3478152aae0f1c0b75b34aa85947R134

Reply:
> I saw that there was a new function `stop` 

I need to remove the SentrySessionReplay.stop function, it's unnecessary. 

> It's a bit confusing to me that this method here is named pause. It would be great to be consistent with the naming.

If I change the pause name to stop, in the integration pause function I would have to call replay stop.
stop function pauses it and also clean it in the next line.